'scripts' is not recognized as an internal or external command,

## 'npm install' returns " 'scripts' is not recognized as an internal or external command," on **windows**

`
` >notes@1.6.3-dev postinstall D:\mypath\notes
  > scripts/postinstall.sh
"scripts" is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! notes@1.6.3-dev postinstall: `scripts/postinstall.sh`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the notes@1.6.3-dev postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:``

--- a/package.json
+++ b/package.json
@@ -25,7 +25,6 @@
   "repository": "mozilla/notes",
   "scripts": {
     "build": "node scripts/build-locales && web-ext build -s src",
-    "post-install-cmd": "bash ./scripts/post-install.sh",
     "clean": "rimraf web-ext-artifacts 'src/sidebar/vendor/!(.gitkeep)' && npm run postinstall",
     "format": "prettier 'src/**/!(vendor)/*.{css,js}' --single-quote --write",
     "lint": "npm-run-all lint:*",
@@ -33,7 +32,7 @@
     "lint:webext": "web-ext lint -s src --ignore-files='sidebar/vendor/*.js' --self-hosted",
     "package": "npm run build && mv web-ext-artifacts/*.zip addon.xpi",
     "postformat": "npm run lint:js -- --fix",
-    "postinstall": "scripts/postinstall.sh",
+    "postinstall": "bash ./scripts/postinstall.sh",
     "prebuild": "npm run clean",
     "start": "web-ext run -s src",
     "start-rtl": "WEB_EXT_PREF=general.useragent.locale=ar npm start"

--- a/package.json
+++ b/package.json
@@ -25,6 +25,7 @@
   "repository": "mozilla/notes",
   "scripts": {
     "build": "node scripts/build-locales && web-ext build -s src",
+    "post-install-cmd": "bash ./scripts/post-install.sh",
     "clean": "rimraf web-ext-artifacts 'src/sidebar/vendor/!(.gitkeep)' && npm run postinstall",
     "format": "prettier 'src/**/!(vendor)/*.{css,js}' --single-quote --write",
     "lint": "npm-run-all lint:*",

